requirements.txt

```diff
@@ -17,7 +17,10 @@ WebOb>=1.8.2 # MIT
 stevedore>=1.20.0 # Apache-2.0
 oslo.config>=6.8.0 # Apache-2.0
 oslo.context>=2.22.0 # Apache-2.0
-oslo.db[mysql]>=8.4.0 # Apache-2.0
+# This was oslo.db[mariadb]>=8.4.0, but that doesn't work with custom oslo.db
+# repo in upper-constraints.txt, so we specify that custom repo again here.
+# This can probably be removed again once Octavia is upgraded.
+oslo.db[mysql] @ git+https://github.com/sapcc/oslo.db.git@stable/yoga-m3 # Apache-2.0
```

Comment:
If I understand correctly how loci build installs requirements, this will not work.

I believe the working options are the following:

1.
Add oslo.db.git to custom-requirements.txt:
```
-e git+https://github.com/sapcc/oslo.db.git@stable/yoga-m3
```
Comment out oslo.db[mysql] here

or 

2. Replace oslo.db[mysql] with the following:
```
oslo.db>=8.4.0 # Apache-2.0
PyMySQL>=0.7.6
```